services/QuillLMS/client/app/bundles/PremiumHub/containers/DataExportContainer.tsx

```diff
@@ -0,0 +1,284 @@
+import React from 'react'
+import queryString from 'query-string';
+import * as _ from 'lodash'
+
+import { FULL, restrictedPage, } from '../shared';
+import CustomDateModal from '../components/usage_snapshots/customDateModal'
+import Filters from '../components/usage_snapshots/filters'
+import { CUSTOM } from '../components/usage_snapshots/shared'
+import { Spinner } from '../../Shared/index'
+import { requestGet, } from '../../../modules/request'
+import { unorderedArraysAreEqual, } from '../../../modules/unorderedArraysAreEqual'
+import DataExportTableAndFields from '../components/dataExportTableAndFields';
+
+const filterIconSrc = `${process.env.CDN_URL}/images/icons/icons-filter.svg`
+
+const DataExportContainer = ({ adminInfo, accessType, }) => {
```

Comment:
Given just how much code is duplicated between this file and the UsageSnapshotsContainer, I wonder if it's worth extracting a lot of that shared code into a parent component for both of them. Maybe that's something we could do as a follow-up card.